examples/dual_shim_asl.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#
+# This shell script performs dual shimming for ASL application: one shim config
+# for the labeling and another shim config for the imaging.
+
+# Download example data
+# TODO: include example ASL dataset
+st_download_data testing_data
+
+# Go inside folder
+# TODO
+
+# Convert dcm2nii
+# TODO (if example dataset is DICOM)
+
+# Compute fieldmap
+# TODO
+
+# Create masks: mask1 for ASL labeling region, mask2: imaging region
+# TODO
+
+# Generate a coil profile based on custom coils
+# TODO
+
+# Generate coil profile from SH basis
+st_generate_profile_spherical_harmonics -i sub-example_fieldmap.nii.gz -order 2 -o coil_profile_sh2.nii.gz
```

Comment:
```suggestion
st_generate_profile_spherical_harmonics -i "sub-example_fieldmap.nii.gz" -order "2" -o "coil_profile_sh2.nii.gz"
```

double-quote string literals

Reply:
the purpose of this shell script is to be used as a tutorial: users will see these command lines as what they are supposed to write in their Terminal. I don't want to "overweight" them with additional quotes if these are not required

Reply:
Sure thing!